clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -397,7 +397,7 @@ private void doSend(ClientRequest clientRequest, boolean isInternalRequest, long
                 log.trace("Sending {} {} with correlation id {} to node {}", clientRequest.apiKey(), request,
                         clientRequest.correlationId(), nodeId);
             } else {
-                log.debug("Using older server API v{} to send {} {} with correlation id {} to node {}",
+                log.trace("Using older server API v{} to send {} {} with correlation id {} to node {}",
```

Comment:
We don't need to log this on every request. Perhaps in ApiVersions.update(), we can log in debug level of any request in nodeApiVersions that's older than the version the client has. This way, this is only logged every time a client connects to the broker.